src/workbench/SideBar.tsx

```diff
@@ -244,25 +242,36 @@ const SideBarContents = ({ panes, ...props }: SideBarContentsProps) => {
 };
 
 const Corner = ({ id, ...props }: BoxProps) => (
-  <Box {...props} pointerEvents="none" width="32px" height="32px">
+  <Box
+    {...props}
+    pointerEvents="none"
+    width={`${cornerSize}px`}
+    height={`${cornerSize}px`}
+  >
     <svg
       width="100%"
       height="100%"
-      viewBox="0 0 32 32"
+      viewBox={`0 0 ${cornerSize} ${cornerSize}`}
```

Comment:
Thanks, this is better. I don't particularly like this corner approach, so better ideas very welcome if you have them (separate PR).

Reply:
Agreed to revert the corners but good to keep the constant.

Not perfect that the theme also has a matching constant but we can revisit another time.